### DISCUSSION TITLE
Syncing photos and videos

### DISCUSSION THREAD

**Discussion by timdonovanuk** [MobiusSync/MobiusSync#51] [General]
Hi,

I was shocked to find there is no native iPhone app for Syncthing, pleased to see MobiusSync in existance, and then super disapointed to find there is no easy way to sync iPhone photos and videos.

I had an iPod literally 20 years ago and remember what a pain it was to simply upload an MP3's to Apples obscure and locked down filesystem. Now it's 2021 and iPhone users still cannot easily access their own data...what is Apples ficking problem...

Given that syncing photos and videos accounts for (likely) a super high majority of use cases for a mobile app, is there likely to be any progress on this soon? 

Thanks!

**Comment by SimonPickup** [comment]
We know this is important to users, and we'd love to use it ourselves. Hopefully we'll get to it after incoroprating untrusted (encrypted) files. Tracking in issues #4 and #9.
